packages/test-ovm-full-node/test/create-support.spec.ts

```diff
@@ -0,0 +1,70 @@
+import './setup'
+
+/* External Imports */
+import { add0x } from '@eth-optimism/core-utils'
+import { createMockProvider, deployContract, getWallets } from 'ethereum-waffle'
+import { addHandlerToProvider } from '@eth-optimism/rollup-full-node'
+import { Contract, Wallet } from 'ethers'
+import {
+  getAddress,
+  keccak256,
+  solidityPack
+} from 'ethers/utils'
+
+/* Contract Imports */
+import * as SimpleCreate2 from '../build/SimpleCreate2.json'
+import * as SimpleStorage from '../build/SimpleStorage.json'
+
+const getCreate2Address = (
+  factoryAddress: string,
+  salt: string,
+  bytecode: string
+): string => {
+  const create2Inputs = [
+    '0xff',
+    factoryAddress,
+    salt,
+    keccak256(bytecode)
+  ]
+  const sanitizedInputs = `0x${create2Inputs.map(i => i.slice(2)).join('')}`
+  return getAddress(`0x${keccak256(sanitizedInputs).slice(-40)}`)
+}
+
+describe('Create2', () => {
+  let wallet
+  let simpleCreate2: Contract
+  let provider 
+  const DEFAULT_SALT = '0x1234123412341234123412341234123412341234123412341234123412341234'
+
+  beforeEach(async () => {
+    provider = await createMockProvider()
+    if (process.env.MODE === 'OVM') {
+      provider = await addHandlerToProvider(provider)
+    }
+    const wallets = await getWallets(provider)
+    const wallet = wallets[0]
+    simpleCreate2 = await deployContract(wallet, SimpleCreate2, [])
+  })
+
+  // TODO uncomment this once ovmCREATE2 is fixed!
```

Comment:
Very nitpick change but could consider 1) updating this TODO comment to be a bit more descriptive (ie. what exactly should be fixed?), and then 2) use `it.skip('...')` instead of commenting it out so it shows up in the unit tests as a skipped test until we fix it